homework1/task1.js

```diff
@@ -0,0 +1,48 @@
+let carSettings = {
+  ABS: 5,
+  traction: 5,
+  stability: 5,
+};
+
+console.log(carSettings);
+
+// function that updates the settings object
+const setSettings = (value, obj) =>
+  Object.keys(obj).forEach((el) => (obj[el] = value));
+
+// main solution, 3 functions that change the difficulty
+const beginner = function () {
+  setSettings(5, carSettings);
```

Comment:
Here need to avoid object mutation in functions. 
Function should object as argument and change this object